# HG changeset patch
# User sbr <sebastien@brissaud.name>
# Date 1375531713 -7200
#      Sat Aug 03 14:08:33 2013 +0200
# Node ID 4989461eedffa9ab64b545a6a6929140484f078d
# Parent  c084b548bf88aed8721dae1ebe3263fa8008ff70
-Fix (r15883): In SignListWindow, right pad the 'None' string

diff --git a/src/signs_gui.cpp b/src/signs_gui.cpp
--- a/src/signs_gui.cpp
+++ b/src/signs_gui.cpp
@@ -198,7 +198,7 @@ struct SignListWindow : Window, SignList
 				uint y = r.top + WD_FRAMERECT_TOP; // Offset from top of widget.
 				/* No signs? */
 				if (this->vscroll->GetCount() == 0) {
-					DrawString(r.left + WD_FRAMETEXT_LEFT, r.right, y, STR_STATION_LIST_NONE);
+					DrawString(r.left + WD_FRAMETEXT_LEFT, r.right - WD_FRAMETEXT_RIGHT, y, STR_STATION_LIST_NONE);
 					return;
 				}
 
